Code/Framework/AzCore/AzCore/Math/UuidSerializer.cpp

```diff
@@ -53,7 +54,7 @@ namespace AZ
 
         Uuid* valAsUuid = reinterpret_cast<Uuid*>(outputValue);
 
-        if (IsExplicitDefault(inputValue))
+        if (IsExplicitDefault(inputValue) || (inputValue == m_zeroUuidString.c_str()))
```

Comment:
There are 4 forms that UUIDs can be written in (with/out braces and with/out the dash). Of those 4 the Json Serialization supports with and without dashes as it needs the braces to identify where the uuid begins/ends as UUIDs can be embedded in a string. There should be two strings to compare with:
```
m_zeroUuidString(AZ::Uuid::CreateNull().ToString<AZStd::string>())
m_zeroUuidStringNoDashes(AZ::Uuid::CreateNull().ToString<AZStd::string>(true, false))
...
if (IsExplicitDefault(inputValue) || (inputValue == m_zeroUuidString.c_str()) || (inputValue == m_zeroUuidStringNoDashes.c_str()))
```
Sorry, forgot to mention this earlier. :(

Reply:
Ok, fixing!